src/app/components/tfrs/tfrs.component.html

```diff
@@ -1 +1,92 @@
-<p>tfrs works!</p>
+<div>
+    <div>
+      <mat-form-field appearance="fill">
+        <mat-label>Vendor</mat-label>
+        <mat-select [(ngModel)]="selectedVendorName">
+          <mat-option>None</mat-option>
+          <mat-option *ngFor="let vendor of vendors" [value]="vendor.name">{{vendor.name}}</mat-option>
+        </mat-select>
+      </mat-form-field>
+
+      <mat-form-field appearance="fill">
+        <mat-label>Status</mat-label>
+        <mat-select [(ngModel)]="selectedStatus">
+          <mat-option>None</mat-option>
+          <mat-option *ngFor="let status of statusList" [value]="status">{{status}}</mat-option>
+        </mat-select>
+      </mat-form-field>
+
+      <mat-form-field appearance="fill">
+        <mat-label>Project Start After</mat-label>
+        <input matInput [matDatepicker]="picker1" [formControl]="startAfterDate" >
+        <mat-hint>MM/DD/YYYY</mat-hint>
+        <mat-datepicker-toggle matIconSuffix [for]="picker1"></mat-datepicker-toggle>
+        <mat-datepicker #picker1></mat-datepicker>
+      </mat-form-field>
+
+      <mat-form-field appearance="fill">
+        <mat-label>Project End Before</mat-label>
+        <input matInput [matDatepicker]="picker2" [formControl]="endBeforeDate">
+        <mat-hint>MM/DD/YYYY</mat-hint>
+        <mat-datepicker-toggle matIconSuffix [for]="picker2"></mat-datepicker-toggle>
+        <mat-datepicker #picker2></mat-datepicker>
+      </mat-form-field>
+
+        <div></div>
+
+      <section>
+        <div class="example-button-row">
+          <button mat-raised-button color="primary" (click)="getProjects()">Search</button>
+        </div>
+      </section>
+    </div>
+</div>
+
+
+<div class="example-container mat-table">
+
+<table mat-table [dataSource]="projectList" matSort (matSortChange)="announceSortChange($event)"  class="mat-elevation-z8 mat-table">
+    
+  <ng-container matColumnDef="name">
+    <th mat-header-cell *matHeaderCellDef mat-sort-header> Name </th>
+    <td mat-cell *matCellDef="let element"> {{element.name}} </td>
```

Comment:
element is not correct name for variable for example new dev wanted to understand what element is referring to they need to understand complete code instead if you see you are running on project list and you need to say project instead of element